cg/apps/lims/limsjson.py

```diff
@@ -19,7 +19,7 @@
 def get_project_type(samples: [dict]) -> str:
     """Determine the project type."""
 
-    data_analyses = set(sample["data_analysis"].lower() for sample in samples)
+    data_analyses = set(sample.get("data_analysis", "mip-dna").lower() for sample in samples)
```

Comment:
Pipeline value here?

Reply:
I'm working on adding Balsamic, I'll add it in that PR